tests/view/test_reservations.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+import pytest
+from timeless.customers.models import Customer
```

Comment:
@paulodamaso according to PEP8 should be one blank line between app modules and thirty-part modules